src/stars.css

```
@@ -0,0 +1,23 @@
+
+.card__stars {
```

it should be separate BEM block `stars`
```suggestion
.stars {
```